Linking error building on OSX

make results in a linking error.

...
Undefined symbols for architecture x86_64:
  "qtsdl::extract_native_context(SDL_Window*)", referenced from:
      qtsdl::GuiRendererImpl::GuiRendererImpl(SDL_Window*) in gui_renderer_impl.cpp.o
ld: symbol(s) not found for architecture x86_64

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/35907308-linking-error-building-on-osx?utm_campaign=plugin&utm_content=tracker%2F371436&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F371436&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/copying.md
+++ b/copying.md
@@ -82,6 +82,7 @@ _the openage authors_ are:
 | Sebastian Brodehl           | sbrodehl                    | sbrodehl@students.uni-mainz.de        |
 | Gaith Hallak                | ghallak                     | gaithhallak@gmail.com                 |
 | Pierre Hallot               | Hallot                      | hallotpierre@gmail.com                |
+| Vicken Simonian             | vsimon                      | vsimon@gmail.com                      |
 
 If you're a first-time commiter, add yourself to the above list. This is not
 just for legal reasons, but also to keep an overview of all those nicknames.

--- a/libopenage/gui/guisys/private/platforms/context_extraction_cocoa.mm
+++ b/libopenage/gui/guisys/private/platforms/context_extraction_cocoa.mm
@@ -9,8 +9,7 @@
 #include "SDL_syswm.h"
 #import <AppKit/NSWindow.h>
 
-namespace openage {
-namespace gui {
+namespace qtsdl {
 
 std::tuple<QVariant, WId> extract_native_context(SDL_Window *window) {
 	assert(window);
@@ -33,4 +32,4 @@ std::tuple<QVariant, WId> extract_native_context(SDL_Window *window) {
 	return std::make_tuple(QVariant::fromValue<QCocoaNativeContext>(QCocoaNativeContext(current_context)), reinterpret_cast<WId>(view));
 }
 
-}} // namespace openage::gui
+} // namespace qtsdl

